### PR TITLE
Map name label

### PR DESCRIPTION
**Pull request by andreus791**
image

### CODE REVIEW

**Comment by lewinjh** [comment]
It makes the interface look too crowded in my opinion.

**Comment by Kromster80** [comment]
As suggested on Trello:
- song name could go down between buttons (as it was in KaM iirc). 
- also take into account that whole menu must reasonably fit into smallest game window (1024*720 iirc).

**Comment by andreus791** [comment]
image

**Comment by andreus791** [comment]
default
default

Limited map name length (.InCastle 1.3 (Bonus)_ Barb_Balanced_77FE8E32 --> .InCastle 1.3 (Bonus)_), dynamic interface to fit autowrapped track names.

**Comment by Kromster80** [comment]
Let's make the texts uniform:

Game time // yellow
hh:mm:ss // white

Map title // yellow
my_map_name // white

Music player // yellow
01_Busy // white

**Comment by andreus791** [comment]
Done, looks better now

image

File: src/gui/KM_InterfaceDefaults.pas [hunk 1]
```diff
@@ -56,6 +56,9 @@   TKMUserInterfaceCommon = class
   OPT_SLIDER_MIN = 0;
   OPT_SLIDER_MAX = 20;
   MAX_SAVENAME_LENGTH = 50;
+  //Limit names length to fit interface width
+  MAX_MAPNAME_LENGTH = 22;
+  MAX_TRACKNAME_LENGTH = 18;
```
Comment (Kromster80) [inline src/gui/KM_InterfaceDefaults.pas]: Let's move these 2 consts to a module where they are used. No need to make them "globally" visible.
After that PR can be merged )

**Review by Kromster80 — CHANGES_REQUESTED** [review]

**Comment by Kromster80** [comment]
Thanks!

**Pull request merged**